keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/PassThruController.kt

```diff
@@ -32,14 +32,30 @@ class PassThruController(
     @PathVariable("cluster") cluster: String,
     @PathVariable("cloudProvider") cloudProvider: String,
     @PathVariable("region") region: String
-  ): List<ActiveServerGroup> =
+  ): List<Any> =
```

Comment:
Hmm... Is there an opportunity here for us to define a base class so that this could be typed `<T : BaseServerGroup>` instead of `Any`? You could also use the same base type in `ServerGroupCollection`.

Reply:
I took a shot at this in 34a20d1